app/src/sharedTest/java/org/oppia/app/help/HelpFragmentTest.kt

```diff
@@ -0,0 +1,71 @@
+package org.oppia.app.help
+
+import androidx.recyclerview.widget.RecyclerView
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.contrib.DrawerActions.open
+import androidx.test.espresso.contrib.RecyclerViewActions.scrollToPosition
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intended
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.help.faq.FAQActivity
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPosition
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPositionOnView
+import org.oppia.app.testing.NavigationDrawerTestActivity
+
+@RunWith(AndroidJUnit4::class)
+class HelpFragmentTest {
+  @Before
+  fun setUp() {
+    Intents.init()
+  }
+
+  @Test
+  fun openNavigationDrawer_selectHelpMenuInNavigationDrawer_scrollRecyclerViewToZeroPosition_showsFAQSuccessfully() {
+    launch(NavigationDrawerTestActivity::class.java).use {
```

Comment:
Your launcher activity for test cases should be `HelpActivity` instead of `NavigationDrawerTestActivity`.

Also, you can add one more test case, click on `hamburger icon` in `HelpActivity` opens `NavigationDrawer`.